lib/index.js

```diff
@@ -51,10 +52,25 @@ function getImportsToResolve (original, includePaths) {
   return imports
 }
 
+function createTransformersMap (transformers) {
+  if (!transformers) {
+    return {}
+  }
+
+  // return map of extension strings to transformer functions
+  return transformers.reduce((extensionMap, transformer) => {
+    transformer.extensions.forEach((ext) => {
+      extensionMap[ext] = transformer.transform
+    })
+    return extensionMap
+  }, {})
+}
+
```

Comment:
I would prefer a single return and to switch the logic of the `transformers` condition.

e.g.

```
function createTransformersMap (transformers) {
    let result = {};

    if (transformers) {

        // return map of extension strings to transformer functions
        result = transformers.reduce((extensionMap, transformer) => {
            transformer.extensions.forEach((ext) => {
                extensionMap[ext] = transformer.transform
            })

            return {};

        }, {})

    };

    return result;
 
 }
```